test/bigtest/helpers/getColumnIndexMapping.js

```diff
@@ -0,0 +1,11 @@
+import { DEFAULT_JOB_LOG_COLUMNS } from '../../../src/utils/constants';
+
+export const getColumnIndexMapping = (
+  columns = DEFAULT_JOB_LOG_COLUMNS,
```

Comment:
It does not make sense to place it on a new line just for one argument. I would also suggest removing the `DEFAULT_JOB_LOG_COLUMNS` as this method has nothing to do with logs.
```suggestion
  columns = [],
```